addons/account/models/account_bank_statement.py

```diff
@@ -507,6 +507,7 @@ class AccountBankStatementLine(models.Model):
     # == Business fields ==
     move_id = fields.Many2one(
         comodel_name='account.move',
+        auto_join=True,
```

Comment:
This is breaking the stable policy, but I recon it is justified with the speedup provided.

It has already been used on our prod without any issues too.